setup.py

```diff
@@ -1,22 +1,62 @@
+import os
 from setuptools import setup, find_packages
-from cli import constants
+from aztk_cli import constants
 from aztk import version
 
+data_files = []
 
-setup(name='aztk',
-      version=version.__version__,
-      description='Utility for data engineers or platform developers to Run distributed jobs in Azure',
-      url='<tbd>',
-      author='Microsoft',
-      author_email='jiata@microsoft.com',
-      license='MIT',
-      packages=find_packages(),
-      scripts=[
-      #     'bin/aztk',
-      ],
-      entry_points=dict(
-            console_scripts=[
-                  "{0} = cli.entrypoint:main".format(constants.CLI_EXE)
-            ]
-      ),
-      zip_safe=False)
+
+def find_package_files(root, directory, dest=""):
+    paths = []
+    for (path, _, filenames) in os.walk(os.path.join(root, directory)):
+        for filename in filenames:
+            paths.append(os.path.relpath(os.path.join(dest, path, filename), root))
+    return paths
+
+
+with open('README.md') as fd:
+    long_description = fd.read()
+
+setup(
+    name='aztk',
+    version=version.__version__,
+    description=
+    'Utility for data engineers or platform developers to Run distributed jobs in Azure',
```

Comment:
Shouldn't this match the repo's tagline? "On-demand, Dockerized, Spark Jobs on Azure (powered by Azure Batch)"

Reply:
yeah didn't touch that one, could change it